Hw4/Rational.cs

```diff
@@ -0,0 +1,78 @@
+public class Rational : IComparable
+{
+    public int N {get; private set;}
+    public int M {get; private set;}
+
+    public Rational(int n, int m)
+    {
```

Comment:
m can be < 0, its normal. Just process this case -> make it positive and change sign of N